packages/text-input-react/src/TextField.tsx

```diff
@@ -58,9 +69,10 @@ export const TextField = ({
                 readOnly={readOnly}
                 value={value}
                 maxLength={maxLength}
+                style={getLength() ? { width: getLength() } : undefined}
                 {...rest}
             />
-            <SupportLabel helpLabel={helpLabel} errorLabel={errorLabel} forceCompact={forceCompact} />
+            {!inline && <SupportLabel helpLabel={helpLabel} errorLabel={errorLabel} forceCompact={forceCompact} />}
```

Comment:
Do we want inline to be it's own component?

I think we shouldn't have props that kills other props without warning, so at least if we do this we should throw a warning or error while in DEV that inline and help/error do not match

Reply:
Yeah, that is a possibility to consider. In that case we might want to move the proposed `ActionTextField` into this package as well, rather than have three separate pkgs...

Reply:
I support the idea of extracting inline as a component and including ActionTextField (and ActionIcon?) in this package as well.

Reply:
As long as we're keeping to just a handful of icons, I agree that thay can live here until we use them in another input type (e.g. select) as well.